### PR TITLE
Treat 0 as valid increment

### PR DESCRIPTION
**Pull request by renatomariscal**
### Description
Current the library has a bug:

A `increment(0)` is treated as `increment()`, so it reports `1` instead of `0`.

### CODE REVIEW

File: lib/statsd.js [hunk 1]
```diff
@@ -91,7 +91,11 @@ Client.prototype.timing = function (stat, time, sampleRate, tags, callback) {
  * @param callback {Function=} Callback when message is done being delivered. Optional.
  */
 Client.prototype.increment = function (stat, value, sampleRate, tags, callback) {
-  this.sendAll(stat, value || 1, 'c', sampleRate, tags, callback);
+  if(value === undefined || value === null || value === null) {
```
Comment (aliakb) [inline lib/statsd.js]: The last two checks are identical.

**Review by aliakb — DISMISSED** [review]
Just one minor thing.

**Review by pshokeen-hbo — APPROVED** [review]
redundant check needs to be fixed, but LGTM otherwise

File: lib/statsd.js (hunk 1 above)
Comment (renatomariscal) [reply lib/statsd.js]: oops, extra copy pasting 😹

**Review by pshokeen-hbo — APPROVED** [review]

**Review by aliakb — APPROVED** [review]

**Pull request merged**